test/e2e/tests/tokens/custom-token-add-approve.spec.js

```diff
@@ -89,7 +89,7 @@ describe('Create token, approve token and approve token without gas', function (
 
         await driver.findClickableElement('#deployButton');
         // approve token from dapp
-        await driver.clickElement({ text: 'Approve Tokens', tag: 'button' });
+        await driver.clickElement('#approveTokens');
```

Comment:
just curious if is there any reason for changing the selector

Reply:
yes, i found that we have an ID to use in dapp page, it is more reliable than text selector in case one day the text is changed.